### PR TITLE
Build Analysis landing page

### PR DESCRIPTION
**Pull request by AlitzelMendez**
https://github.com/dotnet/arcade/issues/9267

### CODE REVIEW

**Review by missymessa — DISMISSED** [review]
LGTM shipit

File: Documentation/Projects/Build Analysis/LandingPage.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Build Analysis
+
+The Build Analysis offers a set of features to make build and pull request results more actionable. 
+
+Explore all the features offered by build analysis:
+
+- [Build Analysis GitHub Check](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/Introduction.md)
```
Comment (riarenas) [inline Documentation/Projects/Build Analysis/LandingPage.md:7]: Would it be worth it to add tiny snippet for what each of these features does? Like 1-2 sentences to spark interest in the link?

**Review by riarenas — DISMISSED** [review]

File: Documentation/Projects/Build Analysis/LandingPage.md (hunk 1 above)
Comment (AlitzelMendez) [reply Documentation/Projects/Build Analysis/LandingPage.md]: I added a small description :D

File: Documentation/Projects/Build Analysis/LandingPage.md [hunk 2]
```diff
@@ -4,8 +4,8 @@ The Build Analysis offers a set of features to make build and pull request resul
...
 
-- [Build Analysis GitHub Check](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/Introduction.md)
-- [Known Issues](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md)
-- [Automatic Test Retries](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/Test%20Retry%20Documentation.md)
-- [Automatic Build Retries](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/BuildRetryOnboard.md)
-- [Test Reporting Queries](https://github.com/dotnet/arcade/blob/a264eb13fea14125f3ef8d4056586cd66fa55309/Documentation/AzureDevOps/TestReportingQueries.md)
+- [Build Analysis GitHub Check](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/Introduction.md): Makes GitHub pull request results more actionable by highlighting build and test information most helpful to .NET developers.
+- [Known Issues](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md): Report known issues so these can be automatically indetified and reported on failed builds.
```
Comment (riarenas) [inline Documentation/Projects/Build Analysis/LandingPage.md:8]:
```suggestion
- [Known Issues](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/KnownIssues.md): Report known issues so these can be automatically identified and reported on failed builds.
```

File: Documentation/Projects/Build Analysis/LandingPage.md [hunk 3]
```diff
@@ -4,8 +4,8 @@ The Build Analysis offers a set of features to make build and pull request resul
...
+- [Automatic Test Retries](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/Test%20Retry%20Documentation.md): Configure tests in your repo's pull requests to automatically retry when they encounter specific errors.
+- [Automatic Build Retries](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/BuildRetryOnboard.md): Configure your repo's pull request builds to automatically retry when it encounters specific errors.
```
Comment (riarenas) [inline Documentation/Projects/Build Analysis/LandingPage.md:10]:
```suggestion
- [Automatic Build Retries](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/Build%20Analysis/BuildRetryOnboard.md): Configure your repo's pull request builds to automatically retry when they encounter specific errors.
```

**Review by riarenas — COMMENTED** [review]
I really like this. Just a couple of minor typos.

**Review by riarenas — APPROVED** [review]

**Pull request merged**